core/src/Cuda/Kokkos_Cuda_Locks.hpp

```diff
@@ -54,7 +54,7 @@
 #include <Cuda/Kokkos_Cuda_Error.hpp>
 
 #ifdef KOKKOS_ENABLE_IMPL_DESUL_ATOMICS
-#include <desul/atomics/Lock_Array_Cuda.hpp>
+#include <desul/atomics/Lock_Array_CUDA.hpp>
```

Comment:
Why is the name different? In master desul the name is still `Lock_Array_Cuda.hpp`: https://github.com/desul/desul/blob/master/atomics/include/desul/atomics/Lock_Array_Cuda.hpp

Reply:
You are right.  I will fix on the desul side.  Thank you.

Reply:
Updated